app/views/components/_search_bar.html.erb

```diff
@@ -2,10 +2,16 @@
 
 <div class="search-bar-container">
   <div class="search-bar">
-    <form action="/" id="search-form" method="get" accept-charset="utf-8">
-      <div class="search-bar-all">
+    <form action="/places" id="search-form" method="get" accept-charset="utf-8">
+      <div class="search-bar-all" data-controller="places-search">
         <%= icon("fas", "search") %>
-        <input class="search-bar-input" type="text" placeholder="Genre, Artist, Track, Address...">
+        <input class="search-bar-input"
+               type="search"
+               placeholder="Genre, Artist, Track, Address..."
```

Comment:
Maybe we should make this consistent with what we propose now which is only "address" or "location".

Reply:
Is it what we will propose though?
Won't we add tags?